build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/release/ReleaseToolsPlugin.java

```diff
@@ -47,6 +47,9 @@ public void apply(Project project) {
 
         final Version version = VersionProperties.getElasticsearchVersion();
 
+        project.getTasks()
```

Comment:
Are these changes intended to be included in this PR?

Reply:
Not now #103335 has been merged (this branch is based on that one)